page.html

```diff
@@ -1,30 +1,34 @@
-
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <link href="fun.css" rel="stylesheet" type = "text/css">
-    <script type="text/javascript" src="fun.js"> </script>
-
-    <title>Timer</title>
-  </head>
-  <body>
-
-<p id="timer">00:00:00</p>
-
-<button onclick="start()">Start</button> 
-<button onclick="stop()">Stop</button>
-<button onclick="reset()">Reset</<button>
-
-
-
-
-
-
-
-
-
 
+<head>
+  <meta charset="utf-8">
```

Comment:
Good habit is to explicitly close self-closing tags.
`<meta charset="utf-8" />`